README.md

```diff
@@ -1,70 +1,127 @@
-# Getting Started with Create React App
+# 📖 Capstone-React-Redux <a name="about-project"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+**Capstone-React-Redux** is  a project which is about building a mobile web application to check a list of metrics data (numeric values) from an API chosen using React and Redux.
 
-## Available Scripts
+I selected a CryptoCurrency API that provides numeric data about all popular cryptocurrencies. The webapp had two pages:
 
-In the project directory, you can run:
+  **-Home page :**
 
-### `npm start`
+1. When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
+2. Along with the category name, it will display some numeric values per category that come from the API.
+3. When the user clicks (or taps) on a category item, the application navigates to the details page.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+**-Details page**
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+1. In the details page, the web app retrieves data from the API to show detailed data in the category.
+2. When the user clicks on the "Back" button (<), the user navigates to the home page.
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## 🛠 Built With <a name="built-with"></a>
 
-### `npm run build`
+### Tech Stack <a name="tech-stack"></a>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux toolkit</a></li>
+  </ul>
+</details>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<!-- Features -->
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Key Features <a name="key-features"></a>
 
-### `npm run eject`
+- **Fetch Cryptocurrencies[Render Detail Pages Dynamically]**
+- **Fetch all data corresponding to a cryptocurrency from corresponding API**
+- **Filtering by different coins**
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## Overview
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+The Loom video Link : https://www.loom.com/share/e4b38fb8217245fb94b363cb6f302a44 
```

Comment:
* Kindly fulfill these requirements below 👇 as your video recording doesn't cover all [these requirements ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Once%20you%20have,very%20proud%20of.)

> Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

a description of the project.
a demo of the project features.
you should also highlight some interesting piece of code or something you built that you are very proud of.

Reply:
I am extremely sorry. I put a previous link. I am attaching the loom video corresponding to this project.

https://www.loom.com/share/e4b38fb8217245fb94b363cb6f302a44